examples/bri-3/package.json

```diff
@@ -22,7 +22,9 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "test:e2e": "jest --config ./test/jest-e2e.json",
+    "snarkjs:ptau": "./zeroKnowledgeKeys/scripts/ptau.sh",
```

Comment:
is zeroKnowledgeKeys folder name descriptive and precise?

Reply:
The folder is for storing the proving and verification keys for the circuits. Do you have any alternative names for it?